types/google-apps-script/google-apps-script.cache.d.ts

```diff
@@ -48,11 +48,12 @@ declare namespace GoogleAppsScript {
      * The data you write to the cache is not guaranteed to persist until its expiration time. You
      * must be prepared to get back null from all reads.
      */
-    interface CacheService {
-      getDocumentCache(): Cache | null;
-      getScriptCache(): Cache | null;
-      getUserCache(): Cache | null;
+    export interface CacheService {
```

Comment:
Nice to meet you.

This change seems to be the problem this issue wants to solve.
I think it would be better not to fix it.
How about that?
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40216